### PR TITLE
[PM-1509] Accessibility for <code> elements

### PR DESCRIPTION
**Pull request by rr-bw**
## Type of change

```
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Improves contrast of `<code>` elements to meet accessibility requirements of a 4.5:1 contrast.

## Code changes

- `variables.scss` - adjust colors for each client/theme

### CODE REVIEW

**Review by danielleflinn — CHANGES_REQUESTED** [review]
@rr-bw thanks for working on this. Design had flag this for web in [PM-621](https://bitwarden.atlassian.net/browse/PM-621). Can you update the web code variables to the variables below (copied from PM-621)?

Light: `#C01176`
Dark: `#F08DC7`

If these also work for any of the other client themes, it would be good to keep them consistent.

**Comment by rr-bw** [comment]
@danielleflinn My bad, I missed that. I've updated those two values across clients. For the Nord theme `#F08DC7` just barely fails accessibility requirements, so I stuck with `#FFA1CC`. Let me know if you'd like something different.

**Review by danielleflinn — APPROVED** [review]
Great! Thank you for checking those against Nord;` #FFA1CC` will be fine.

**Review by jlf0dev — DISMISSED** [review]

**Review by cagonzalezcs — DISMISSED** [review]
Changes look good, I see some other hardcoded values. Down the line, I wonder if it makes sense to also set this up as referenceable variables

**Review by Hinton — CHANGES_REQUESTED** [review]
We should sync the tailwind and scss variables.

File: apps/web/src/scss/variables.scss [hunk 1]
```diff
@@ -108,6 +108,9 @@ $darkDarkBlue2: #1f242e;
 $darkInputColor: $white;
 $darkInputPlaceholderColor: $darkGrey1;
 
+$code-color: #c01176;
+$code-color-dark: #f08dc7;
```
Comment (Hinton) [inline apps/web/src/scss/variables.scss:112]: The variable names aren't aligned with the convention in this file. The light color should be under the light variables.
```suggestion
$lightCodeColor: #c01176;
$darkCodeColor: #f08dc7;
```

**Review by Hinton — APPROVED** [review]

**Pull request merged**